### PR TITLE
git: add git pull --rebase

### PR DESCRIPTION
**Pull request by herbygillot**

### CODE REVIEW

File: git [hunk 1]
```diff
@@ -148,6 +148,10 @@ git diff --no-index path/to/file/A path/to/file/B
 git fetch --all
 git reset --hard origin/master
 
+# To pull a remote branch but rebase any locally differing comments onto the
```
Comment (vidosits) [inline git:151]: "comments" here is a typo for "commits", right?

Comment (herbygillot) [reply git]: You are correct, not sure how I missed that. Fixed.

**Comment by chrisallenlane** [comment]
Thanks, guys!

**Comment by herbygillot** [comment]
Thank you

**Pull request merged**